app/components/elements/tab_form/tab_list_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= tag.div class: classes, 'data-controller': 'tab-error' do %>
   <div class="col-md-3">
-    <%= tag.div class: 'nav nav-pills d-flex flex-md-column px-2 py-1', role: 'tablist', 'aria-orientation': 'vertical' do |component| %>
+    <%= tag.div class: 'nav nav-pills d-flex flex-md-column rounded-top rounded-bottom', role: 'tablist', 'aria-orientation': 'vertical' do |component| %>
```

Comment:
Round the corners of the surrounding div and remove padding that breaks the design.